### PR TITLE
Feature/delegations and mentor

### PR DESCRIPTION
**Pull request by aliceringstromsk**
## Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Content/Data

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [x] No
      
## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (if applicable).
- [ ] I have added/updated tests to cover my changes (if applicable).

### CODE REVIEW

File: frontend/src/pages/[userId]/index.tsx [hunk 1]
```diff
@@ -34,21 +35,27 @@ export const CheckList: React.FC = () => {
...
     loaded,
-    loading,
     refresh: refreshChecklist,
   } = useChecklist((query?.userId as string) || username);
+  const { data: delegatedChecklists, refresh: refreshDelegatedChecklists } = useDelegatedChecklists();
 
   const managedChecklist = managedChecklists.filter((employee) => employee.employee.username === query?.userId)[0];
-  const data = currentView === 0 ? managedChecklist : employeeChecklist;
+  const delegatedChecklist = delegatedChecklists.filter((employee) => employee.employee.username === query?.userId)[0];
+  const data =
```
Comment (jerkerosatiesk) [inline frontend/src/pages/[userId]/index.tsx:50]: Får felmeddelande när datat ska loopas.
`Cannot read properties of undefined (reading 'map')`

**Review by jerkerosatiesk — CHANGES_REQUESTED** [review]

File: frontend/src/components/assign-mentor-modal/assign-mentor-modal.component.tsx [hunk 2]
```diff
@@ -47,26 +55,42 @@ export const AssignMentorModal: React.FC = () => {
...
-        : <p className="text-small cursor-pointer">
-            <Link onClick={openHandler}>
-              <Icon name="plus" size="1.5rem" className="mr-4" />
-              Lägg till mentor
-            </Link>
-          </p>
-        }
+        : isManager ?
+          <Button variant="tertiary" onClick={openHandler} size="sm" className="mt-8">
+            Lägg till mentor
```
Comment (jerkerosatiesk) [inline frontend/src/components/assign-mentor-modal/assign-mentor-modal.component.tsx:75]: Språkfil?

File: frontend/src/components/checklist-sidebar/checklist-sidebar.component.tsx [hunk 3]
```diff
@@ -40,55 +42,68 @@ export const ChecklistSidebar: React.FC = () => {
...
+                  <strong>{t('checklists:delegated_employees')}</strong>
+
+                  {data.delegatedTo?.map((email, index) => {
+                    return (
+                      <div key={index} className="flex justify-between mb-16 mt-8">
+                        <div>
+                          <Avatar size="sm" className="mr-4" rounded /> {email}
+                        </div>
+
+                        <Button
```
Comment (jerkerosatiesk) [inline frontend/src/components/checklist-sidebar/checklist-sidebar.component.tsx:84]: Knappen ser ej rätt ut?
tertiary med showbackground=false?

**Review by jerkerosatiesk — APPROVED** [review]
Om name-attributen fyller någon funktion så är det ok

**Review by hensankom — APPROVED** [review]

**Pull request merged**